presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -126,6 +134,8 @@ The following types are supported:
 
     Returns the smallest of the provided values.
 
+.. _all_any_some_predicates:
```

Comment:
Let's call this by the proper name `quantified_comparison_predicates`

Reply:
Done